app/controllers/Challenge.scala

```diff
@@ -166,7 +173,7 @@ final class Challenge(
         case Some(c) => api.cancel(c) inject jsonOkResult
         case None =>
           api.activeByIdFor(id, me) flatMap {
-            case Some(c) => api.decline(c) inject jsonOkResult
+            case Some(c) => api.decline(c, None) inject jsonOkResult
```

Comment:
Is there a reasonable `reason` we could specify for all declines triggered by `cancel`? Not familiar enough with challenges to come up with a good guess here.